README.md

```diff
@@ -157,6 +157,22 @@ It is possible to list all of the versions of `qgis` available on your platform
 conda search qgis --channel conda-forge
 ```
 
+To start QGIS, call:
+
+```
+qgis
+```
+
+**Notes for macOS**
+
+If the QGIS window remains invisible ([issue #183](https://github.com/conda-forge/qgis-feedstock/issues/183)), you might need to set: 
+```
+export QT_MAC_WANTS_LAYER=1
```

Comment:
This will be overwritten in the next re-render. The README is auto-generated and changes like this won't persist. We don't have a solid way to document package peculiarities like this one. Maybe open an issue and leave it open?

Reply:
Not sure if it's visible enough, but we could enable the discussions tab for the repository.

Reply:
Intersting way to update a readme. How can you edit the README.md file persistently at all?